src/esm_environment/esm_environment.py

```diff
@@ -476,7 +470,7 @@ def get_shell_commands(self):
 
         environment = []
         # Write module actions
-        if "module_actions" in self.config:
+        if self.config.get("module_actions") is not None:
```

Comment:
Talking about style, this is the only place in the code where we have this syntax. Normally we would do:
```
if self.config.get("module_actions"):
```
because `get` if there is not second argument, returns `None` if the variable is not found, and the `if` condition is therefore not satisfied. Is there a specific reason why we want this syntax here?

Reply:
There is no specific reason. It may become relevant if you provide something like an empty list or empty dict, but for module actions (which should, if I remember correctly, always give back a list) it does not matter.

Reply:
Good point. So this will stay as it is now.